src/evaluation/evaluation-error-handler.service.ts

```diff
@@ -124,7 +124,9 @@ export class EvaluationErrorHandlerService {
 
       // Email subject
       const processCode = currentEvaluationQueue?.processCode || 'N/A';
-      const emailSubject = `${processCode} Evaluation Feedback for ORIS code ${emailTemplateContext.orisCode} Unit ${emailTemplateContext.configuration}`;
+      const env = this.configService.get<string>('app.env')?.trim()?.toLowerCase();
+      const subjectSuffix = env && !['prod', 'production', ''].includes(env) ? ` (sent from ECMPS 2.0 ${env})` : '(sent from ECMPS 2.0)';
```

Comment:
The final template already has a space, the space before the parenthesis here is extra:
```suggestion
      const subjectSuffix = env && !['prod', 'production', ''].includes(env) ? `(sent from ECMPS 2.0 ${env})` : '(sent from ECMPS 2.0)';
```